components/support.mdx

```diff
@@ -1,7 +1,7 @@
 Clearpath is committed to your success.
 Please get in touch with us and we will do our best to get you rolling again quickly: <support@clearpathrobotics.com>.
 
-To get in touch with a salesperson regarding Clearpath Robotics products, please email <sales@clearpathrobotics.com>.
+To get in touch with a salesperson regarding Clearpath Robotics products, please email <research-sales@clearpathrobotics.com>.
 
 If you have an issue that is specifically about ROS and is something which may be of interest to the broader community, consider asking it on <https://answers.ros.org>.
```

Comment:
ROS answers is no longer active, https://robotics.stackexchange.com/ is the new site.

Reply:
Thanks, I have updated the link